### ISSUE TITLE
magic number ignoreNamedArgument not working with inheriting abstract class

### ISSUE THREAD

**Issue by cyfung** [arturbosch/detekt#992]
```
abstract class A(n: Int)

object B : A(n = 5)
```

## Expected Behavior
magic number should not be reported if ignoreNamedArgument is true

## Current Behavior
magic number is reported even if ignoreNamedArgument is true

**Comment by arturbosch** [comment]
Thanks for reporting.
I could reproduce this with following test case:
```
			it("should ignore named arguments in inheritance") {
				val code = """
					abstract class A(n: Int)

					object B : A(n = 5)
				""".trimIndent()
				val rule = MagicNumber(TestConfig(mapOf("ignoreNamedArgument" to "true")))
				assertThat(rule.lint(code)).isEmpty()
			}
```
